en/config_mc/README.md

```diff
@@ -1,10 +1,152 @@
+
+If using PWM ESCs and OneShot ESCs (but not DShot and DroneCAN/Cyphal ESC) you should then perform [ESC Calibration](../advanced_config/esc_calibration.md) before proceeding to [Motor Configuration](../config/actuators.md#motor-configuration).
+This ensures that all ESC provide exactly the same output for a given input (ideally we'd calibrate ESCs first, but you can't calibrate your ESCs until outputs are mapped).
+
+The final step is [Motor Configuration](../config/actuators.md#motor-configuration), which sets the directions of motors, and their disarmed, armed and high limits.
+
+Relevant topics:
+
+- [Vehicle (Frame) Selection](../config/airframe.md) - Select vehicle type to match your frame.
+- [Actuator Configuration and Testing](../config/actuators.md) - Vehicle geometry, output mapping, motor configuration, testing.
+- [ESC Calibration](../advanced_config/esc_calibration.md) - Do between output mapping and motor configuration (topic above) for PWM and OneShot ESC.
+
+## Sensor Setup and Calibration
+
+PX4 most commonly relies on a magnetometer (compass) for direction information, a barometer for altitude, a gyroscope for attitude, an accelerometer for velocity calculations, and a GPS/GNSS for global position.
+Pixhawk flight controllers (and many others) have inbuilt magnetometer, accelerometer, gyroscope, and barometer.
+The inbuilt compass usually isn't particularly reliable, and it is common to also add an external compass (usually with a GPS for position).
+
+We first need to set the [Sensor Orientation](../config/flight_controller_orientation.md), informing PX4 how the autopilot (and its inbuilt sensors) and external compasses are oriented relative to the vehicle.
+Generally you'll orient towards the front of the vehicle and not have to set anything.
+Once that is done we need to calibrate the compass(es), gyroscope, and accelerometer.
+
+The core sensor setup is covered in these topics.
+
+- [Sensor Orientation](../config/flight_controller_orientation.md)
+- [Compass](../config/compass.md)
+- [Gyroscope](../config/gyroscope.md)
+- [Accelerometer](../config/accelerometer.md)
+
+PX4 can use other peripherals, such as distance sensors, optical flow sensors, traffic avoidance alarms, cameras, and so on.
+
+- [Flight Controller Peripherals](../peripherals/README.md) - Setup specific sensors, optional sensors, actuators, and so on.
+
+:::note
+Barometers don't need calibration, and if you've mounted the flight controller level, there isn't much value in a [Level Horizon](../config/level_horizon_calibration.md) calibration either.
+
+Obviously you don't need to calibrate or configure anything that isn't present or used by PX4.
+For multicopter, that includes [airspeed sensors](../config/airspeed.md), which are not used for flight control.
+:::
+
+## GPS Setup
```

Comment:
I don't think the GPS requires a dedicated section.

Reply:
Agreed. I'm going to remove this. I will include it above as "a sensor that is not covered"